go-controller/pkg/node/controllers/egressip/egressip.go

```diff
@@ -1,6 +1,7 @@
 package egressip
 
 import (
+	"context"
```

Comment:
`context`: is this import needed?

Reply:
yes, for ln 251.

Reply:
ah the "context" import was in the wrong commit - fixed.